### PR TITLE
Showing help when no input parameters are given and suppress warnings for cmds

### PR DESCRIPTION
**Pull request by parichit**
This change is made in the base.py file to check if the user has provided any arguments or not.

Currently, if the workflows are run without any parameters then there is no help message to prompt the user of what went wrong.

Added the check and put an appropriate error message.

### CODE REVIEW

**Comment by parichit** [comment]
Thanks @serge for pointing out the PEP8 issue. I updated the length of the code (in the base.py file) to conform to the PEP8 formatting style. The PEP8 issue is now resolved.

I am waiting for the Travis-ci checks to complete now.

File: dipy/fixes/argparse.py [hunk 1]
```diff
@@ -1909,6 +1909,10 @@ def consume_positionals(start_index):
         # if we didn't use all the Positional objects, there were too few
         # arg strings supplied.
         if positionals:
+            #  printing user friendly help message to tell about missing
+            #  arguments.
+            print("Too few arguments. Program", self.prog, "expects arguments"
```
Comment (skoudoro) [inline dipy/fixes/argparse.py]: Can you some "\n\n"

Comment (parichit) [reply dipy/fixes/argparse.py]: @skoudoro: Thanks for pointing this out. I have added the line gaps and committed. Thanks.

**Comment by skoudoro** [comment]
ok, LGTM, I will merge tomorrow if there is no other comment

**Pull request merged**